Source/WebCore/platform/network/HTTPParsers.h

```diff
@@ -97,8 +97,8 @@ AtomString extractReasonPhraseFromHTTPStatusLine(const String&);
 WEBCORE_EXPORT XFrameOptionsDisposition parseXFrameOptionsHeader(StringView);
 WEBCORE_EXPORT OptionSet<ClearSiteDataValue> parseClearSiteDataHeader(const ResourceResponse&);
 
-// -1 could be set to one of the return parameters to indicate the value is not specified.
-WEBCORE_EXPORT bool parseRange(StringView, RangeAllowWhitespace, long long& rangeStart, long long& rangeEnd);
+// std::nullopt could be set to one of the return parameters to indicate the value is not specified.
+WEBCORE_EXPORT bool parseRange(StringView, RangeAllowWhitespace, std::optional<size_t>& rangeStart, std::optional<size_t>& rangeEnd);
```

Comment:
Should we change this to use a return value instead of out arguments?

Reply:
Yes, since I'm modernizing this function, I might as well. I'll make this change before landing.